packages/lib/gpcircuits/circuits/proto-pod-gpc.circom

```diff
@@ -210,6 +214,36 @@ template ProtoPODGPC (
         isNullfierHashRevealed <== ownerIsNullfierHashRevealed
                                                                           );
 
+    
+    /*
+     * 1 BoundsCheckModule with its inputs & outputs
```

Comment:
Why structure this as 1 module of variable size rather than MAX_BOUNDS_CHECKS modules?  I feel like the logic might actually be clearer if the MaybeInputSelector could be inside the argument to a singular BoundsCheckModule which is basically the InInterval template.  The for loop in the BoundsCheckModule seems to be largely reversing the work done by the for loop here.